CGPs/cgp-0039.md

```diff
@@ -0,0 +1,98 @@
+---
+cgp: 39
+title: Increase Block Target Density
+date-created: 2021-09-06
+author: Mariano Cortesi (@mcortesi)
+status: DRAFT
+date-executed:
+governance-proposal-id:
+---
+
+## Overview
+
+CGP - Celo Governance Proposal
+
+Pick a four digit number subsequent to the latest PR raised and name your file that with the suffic P (for pending).
+Eg. if the latest PR to the CGP folder was [0034], then your PR should be [0035P].
```

Comment:
please delete lines 15 and 16, see note about adding a simple overview summary of this proposal.